src/ocm/components/clusterWizard/ClusterWizardFooter.tsx

```diff
@@ -33,6 +34,9 @@ const ValidationSection = ({
   const { currentStepId } = useClusterWizardContext();
   const validationsInfo = cluster && selectClusterValidationsInfo(cluster);
   const { t } = useTranslation();
+  const { isPlatformIntegrationSupported } = useClusterSupportedPlatforms(
+    cluster?.id ? cluster?.id : '',
```

Comment:
```suggestion
    cluster?.id ?? '',
```

Reply:
changed it

Reply:
This hook won't work w/o a `cluster.id`, you can assume that `isPlatformIntegrationSupported` will be `false` when you don't have the `cluster.id`.

Reply:
now is better?